### ISSUE TITLE
Codeml does not produce mlc files, it combines with the output

### ISSUE THREAD

**Issue by balasink** [GuangchuangYu/ggtree#228]
Hello!

Codeml does not produce separate mlc files anymore, the data are merged with the general output file defined in codeml.ctl. I was wondering if I could see an actual mlc output file so I could make my own that I can then use with ggtree? I need to plot the dN/dS values along branches and read.codeml does not seem to work with my rst file or general output files.
Thanks!

**Comment by paula-torres** [comment]
Hi, did you solve this issue? I have the same problem

**Comment by kdbchau** [comment]
Nope. Never solved and never got help with it so I just don't use it.
